examples/mediapipe/customMediaPipe/main.ts

```diff
@@ -0,0 +1,95 @@
+import CameraSource from '../../common/js/camera-source'
+import { getVonageEventEmitter, isSupported, MediaPipeModelType } from '@vonage/ml-transformers'
+import {setVonageMetadata, MediaProcessorConnectorInterface, MediaProcessorConnector} from '@vonage/media-processor'
+import MediaProcessorHelperMain from './js/MediaProcessorHelperMain'
+import MediaProcessorHelperWorker from './js/MediaProcessorHelperWorker'
+import { MediapipeMediaProcessorInterface } from './js/MediapipeInterfaces';
+
+async function main() {
+  try {
+    await isSupported();
+  } catch(e) {
+    throw('Something bad happened: ' + e);
```

Comment:
This should be something else/something more (maybe an alert of some VIVID alert message). This way can go unnoticed for the user.